libs/dh/market-participant/actors/feature-actors/src/lib/dh-actors-custom-filter-predicate.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { EicFunction } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import { AllFiltersCombined } from './actors-filters';
+import { DhActor } from './dh-actor';
+import { dhParseJSON } from './dh-json-util';
+
+/**
+ * Custom filter predicate function that runs for each actor in the table.
+ *
+ * @returns `true` if an actor should be shown in the table, `false` otherwise.
+ */
+export const dhActorsCustomFilterPredicate = () => {
```

Comment:
question: why is this a higher-order function?

Reply:
No particular reason. I think I started with it initially, and kept it afterwards. Will get rid of it :)